react-for-atom.js.flow

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
I couldn't get flow to work correctly with the getters. So we'd have to ditch that. I added tests to make sure that flow is behaving correctly.